app.js

```diff
@@ -21,10 +25,18 @@ let error = null;
 
 window.addEventListener('load', async () => {
     // > Part B: Add a click event listener for the todoEl
-    //      - call the async supabase function to delete all todos
-    //        and get the response
+    //      - call the async supabase function to get todos
+    const response = await getTodos();
     //      - set the todos and error state from the response
+    error = response.error;
+    todos = response.data;
     //      - if there's an error call displayError
+    if (error) {
+        displayError();
+    }
+    if (todos) {
+        displayTodos;
```

Comment:
You need to invoke displayTodos with the (). This is why the todos aren't showing up when you load the page.